### PR TITLE
support-monorocksdb

### PR DESCRIPTION
**Pull request by area363**
This PR takes `monorocksdb` as an option for `--store-type`.
After this PR, `monorocksdb` will take the [original rocksdb(MonoRocksDBStore)](https://github.com/planetarium/libplanet/blob/main/Libplanet.RocksDBStore/MonoRocksDBStore.cs) and `rocksdb` will take the [new rocksdb](https://github.com/planetarium/libplanet/blob/main/Libplanet.RocksDBStore/RocksDBStore.cs).

### CODE REVIEW

**Review by riemannulus — APPROVED** [review]

File: Libplanet.Headless/Hosting/LibplanetNodeService.cs [hunk 1]
```diff
@@ -269,6 +269,23 @@ public virtual Task StopAsync(CancellationToken cancellationToken)
                     Log.Error("RocksDB is not available. DefaultStore will be used. {0}", e);
                 }
             }
+            else if (type == "monorocksdb")
+            {
+                 try
```
Comment (moreal) [inline Libplanet.Headless/Hosting/LibplanetNodeService.cs:274]:
```suggestion
                try
```

**Review by moreal — CHANGES_REQUESTED** [review]

**Review by moreal — CHANGES_REQUESTED** [review]
The build seems failed with the below message.

```
Hosting/LibplanetNodeService.cs(276,46): error CS0234: The type or namespace name 'MonoRocksDBStore' does not exist in the namespace 'Libplanet.RocksDBStore' (are you missing an assembly reference?) [/home/runner/work/NineChronicles.Headless/NineChronicles.Headless/Libplanet.Headless/Libplanet.Headless.csproj]
```

It seems there is no `MonoRocksDB` class at [Libplanet.RocksDB in the submodule](https://github.com/planetarium/libplanet/tree/95a8409bf11017d742be14ff19e49b913ceaeb42/Libplanet.RocksDBStore). Does it need to bump submodule? 🤔

**Comment by area363** [comment]
Yes, this will pass after bumping Lib9c to [this branch](https://github.com/planetarium/lib9c/tree/rc-v100036). However, `rc-v100036` hasn't been fixed yet so I'm putting this to hold for now.

**Comment by longfin** [comment]
I've updated lib9c in NineChronicles.Headless at b7c2cddf.

**Review by longfin — APPROVED** [review]

**Comment by longfin** [comment]
Oops, I'll bump lib9c once again... 😢

**Pull request merged**